src/components/EmojiPicker/EmojiPickerButton.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {Pressable} from 'react-native';
+import PropTypes from 'prop-types';
+import styles from '../../styles/styles';
+import * as StyleUtils from '../../styles/StyleUtils';
+import getButtonState from '../../libs/getButtonState';
+import * as Expensicons from '../Icon/Expensicons';
+import Tooltip from '../Tooltip';
+import Icon from '../Icon';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import * as EmojiPickerAction from '../../libs/actions/EmojiPickerAction';
+
+const propTypes = {
+    /** Flag to disable the emoji picker button */
+    isDisabled: PropTypes.bool,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+    isDisabled: false,
+};
+
+class EmojiPickerButton extends React.Component {
```

Comment:
This should be funational...

Reply:
If I was using a function component, I was getting an error for setting `this.emojiPopoverAnchor = el` reference.

Reply:
Oh, I see. Missed the ref. Could you please add a comment over it so nobody touches it?